src/popup/router/components/TabBar.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="tab-bar">
+    <button :class="{ selected: $route.path === '/account' }">
+      <router-link to="/account">
+        <Home />
+      </router-link>
+    </button>
```

Comment:
```suggestion
    <RouterLink to="/account">
      <Home />
    </RouterLink>
```
proposing to drop extra wrapper